database/README.md

```diff
@@ -0,0 +1,36 @@
+# Day 2 Databases and your Go app
+
+This section of the course contains examples and exercises for effectively leveraging a database in the backend of your Web services.. The following are examples of finished exercises for day 2 of the [O'reilly Media Online Learning Course](https://www.oreilly.com/live-events/go-for-web-development-in-3-weeks/0636920091015/). They can be completed by following along with the instruction or independently.
+
+## Exercise 1
+
+Use the appropriate database driver and go module to connect to the provided postgres database. The username and password are provided by the instructor (See below if completing independently). Once you have connected run a query based on the following schema...
+
+TODO: include an ERD or other schema docs.
+
+_NOTE_: If you are completeling this indepently. You can run postgres locally in a docker container using the following command.
+
+```
+docker pull postgres
+docker run -e POSTGRES_PASSWORD=postgres -e POSTGRES_USERNAME=postgres -p 5431:5432 postgres
```

Comment:
use not-postgres for username/password so they aren't accidentally trapped in being correct by default.